### PR TITLE
Restores some functions food had

### PR DESCRIPTION
**Pull request by ArcaneDefence**
##  About The Pull Request

Synthmeat steaks are a subtype of plain so that they can be used in burgers the way they could before
Colored burgers have color again, as they were just the greyscale sprite

## Why It's Good For The Game

image

Fuck you, I had this before, I want it still. 

## Changelog
:cl:
fix: Synthmeat works for burgers again, and colored burgers are colored once more.
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
pretty patties!

File: code/datums/components/storage/food_storage.dm [hunk 1]
```diff
@@ -58,6 +58,9 @@
 		to_chat(user, "<span class='warning'>There's something in \the [parent].</span>")
 		return
 
+	if(istype(inserted_item, /obj/item/pen) && user.a_intent == INTENT_HELP || iscyborg(user)) //Allows renaming and prevents borgs from breaking their modules by hiding them inside foods
```
Comment (Jared-Fogle) [inline code/datums/components/storage/food_storage.dm:61]: Split these into two checks, one for checking pens and the other for checking cyborgs.

**Review by Mothblocks — DISMISSED** [review]

**Comment by Qustinnus** [comment]
The pen and borg module fixes are snowflake remove those and keep the rest.

I'm also confused about the synthsteak, I try not to change the type paths besides the starting but so either plain was never in the type path or I accidentally removed it.

File: code/game/objects/items/food/burgers.dm [hunk 2]
```diff
@@ -182,55 +182,63 @@
 	name = "red burger"
 	desc = "Perfect for hiding the fact it's burnt to a crisp."
 	icon_state = "cburger"
+	color = "#ff0000"
```
Comment (Rohesie) [inline code/game/objects/items/food/burgers.dm:185]: Use color defines

**Comment by Rohesie** [comment]
Conflicts due to #53894

**Comment by ShizCalev** [comment]
Conflicts.

**Pull request merged**